src/site/controller.ts

```diff
@@ -6,10 +6,15 @@
 import { Request, Response, Router } from 'express';
 import { ExpressHandler, HTTPError } from '../utils';
 import { SiteModel } from './model';
+import { verifySiteId } from './utils';
 
 const SiteRouter = Router();
 export default SiteRouter;
 
+SiteRouter.get(['/:siteId'], (res, req, next) =>
+  verifySiteId(res, req, next, 'param'),
```

Comment:
This can be replaced by the more generic verison once my MR is in.

Reply:
done